apps/server/src/modules/sharing/service/share-token.service.ts

```diff
@@ -61,7 +63,13 @@ export class ShareTokenService {
 			case ShareTokenParentType.Task:
 				parentName = (await this.taskService.findById(shareToken.payload.parentId)).name;
 				break;
+			case ShareTokenParentType.ColumnBoard:
+				// TODO - well the column board should have a name...
+				parentName = (await this.columnBoardService.findById(shareToken.payload.parentId)).title;
```

Comment:
I think it's more readable with less overloaded code:
const { title } = await this.columnBoardService.findById(shareToken.payload.parentId);
parentName = title;
or
const { title: parentName } = await...

Reply:
decided to use same style as the other code